### DISCUSSION TITLE
Global Vulnerabilities

### DISCUSSION THREAD

**Discussion by jsmythe** [npm/feedback#645] [CLI] [answered]
Running "npm update -g" shows vulnerabilities, if any, but not where they occur. Can npm be modified to show the vulnerability locations?

**Comment by MylesBorins** [comment, marked as answer]
@jsmythe the vulnerabilities will be in the globally installed package. How are you imagining we could improve the output? For explicit suggestions an issue should be opened on the https://github.com/npm/rfcs repository

**Comment by jsmythe** [comment]
When I do "npm update -g" it shows vulnerabilities, such as

   xx vulnerabilities (xx moderate, xx high, xx critical)

but it does not show where the vulnerabilities occur. Since npm update 
can determine the vulnerabilities and their severity, it should be able 
to display the path where they occur, such as

   xxx/npm/node_modules/xxx/node_modules/xxx/...

On 2/28/2022 12:01 PM, Myles Borins wrote:
>
> @jsmythe <https://github.com/jsmythe> the vulnerabilities will be in 
> the globally installed package. How are you imagining we could improve 
> the output? For explicit suggestions an issue should be opened on the 
> https://github.com/npm/rfcs repository
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/npm/feedback/discussions/645#discussioncomment-2267200>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AG4RIMKXLV2N4YNLAWH2PLLU5OS57ANCNFSM5PLC6ICQ>.
> Triage notifications on the go with GitHub Mobile for iOS 
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> 
> or Android 
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. 
>
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>
>

-- 
This email has been checked for viruses by AVG.
https://www.avg.com
